spec/factories/ingredients.rb

```diff
@@ -0,0 +1,11 @@
+FactoryBot.define do
+  factory :ingredient do
+    name { Faker::Food.ingredient }
+    sku { Faker::Alphanumeric.alphanumeric(number: 10) }
+    price { rand(50..300)*10 }
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Layout/SpaceAroundOperators: Surrounding space missing for operator *.